is_core/forms/utils.py

```diff
@@ -12,5 +15,22 @@ def add_class_name(attrs, class_name):
 class ReadonlyValue:
 
     def __init__(self, value, humanized_value):
-        self.value = value
-        self.humanized_value = humanized_value
+        self._value = value
+        self._humanized_value = humanized_value
+
+    def render(self):
+        if self._humanized_value is None:
+            return format_html(
+                '<p>{}</p>',
+                str(self._value),
```

Comment:
I think the `str` here is not nedded, `format` typically forces string.

Reply:
you are true